include/os/freebsd/spl/sys/random.h

```diff
@@ -45,4 +48,23 @@ random_get_pseudo_bytes(uint8_t *p, size_t s)
 	return (0);
 }
 
+static inline uint32_t
+random_in_range(uint32_t range)
+{
+#if  __FreeBSD_version >= 1300108
+	return (prng32_bounded(range));
```

Comment:
@amotin shouldn't this be #if defined(_KERNEL) && __FreeBSD_version >= 1300108

Reply:
I don't think this file is used in user-space.  It has separate implementation in include/sys/zfs_context.h.  Am I wrong?

Reply:
I am currently doing the merge to main multiple things break. I will send you an e-mail.